nodeadm/api/v1alpha1/nodeconfig_types.go

```diff
@@ -29,8 +30,10 @@ type NodeConfigList struct {
 }
 
 type NodeConfigSpec struct {
-	Cluster      ClusterDetails  `json:"cluster,omitempty"`
-	FeatureGates map[string]bool `json:"featureGates,omitempty"`
+	Cluster      ClusterDetails    `json:"cluster,omitempty"`
+	Labels       map[string]string `json:"labels,omitempty"`
+	Taints       []v1.Taint        `json:"taints,omitempty"`
```

Comment:
these are applied by kubelet, but they are "node" configuration so not sure if they feel best here or in a `Spec.Kubelet.{Labels,Taints}`

Reply:
I think it belongs in `Kubelet`

Reply:
updated 👍